### ISSUE TITLE
Asian voice

### ISSUE THREAD

**Issue by giahuy45** [skiselkov/BetterPushbackC#350]
Hello, Can you let me record some voice for the Asia region for the next update? Thank you

**Comment by sandpatch** [comment]
Hello @giahuy45 
I am working on a project to add more voices. Do you have discord?

**Comment by giahuy45** [comment]
Yeah, that's cool. What is your discord name?

**Comment by sandpatch** [comment]
Sandpatch#6455
Write to me

**Comment by martinlest** [comment]
> I am working on a project to add more voices.
  Anything come of it?

**Comment by sandpatch** [comment]
@martinlest
Yeah, this is what I free for now. https://forums.x-plane.org/index.php?/files/file/65338-betterpushbackc-extra-soundpacks/

**Comment by martinlest** [comment]
Right, thank you.
